app/models/post.rb

```diff
@@ -3,15 +3,19 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
```

Comment:
- @PeteUgwu great job for making sure that title of the post must be present :+1: Once more, let's kindly add another test for the length of the title. The title must not exceed 250 characters.